aries_cloudcontroller/controllers/proof.py

```diff
@@ -80,9 +80,11 @@ async def get_presentation_credentials(
         )
 
     # Send a problem report for presentation exchange
-    async def send_problem_report(self, pres_ex_id, request):
+    async def send_problem_report(self, pres_ex_id, explanation: str):
+        explanation_request = {"explain_ltxt": explanation}
```

Comment:
Don't know that this should be called explanation_request.  Maybe just body even would be fine

Reply:
I've checked the code and the only other place regarding Problem_report is in issuer.py where there already is an explanation param that builds the body like that.